circuits/pedersen_tree/src/main.nr

```diff
@@ -15,28 +14,40 @@ fn main(
   nullifierHash: pub Field, 
 ) -> pub Field {
     // Compute public key from private key to show ownership
-    let pubkey = std::scalar_mul::fixed_base(priv_key);
-    let pubkey_x = pubkey[0];
-    let pubkey_y = pubkey[1];
+    let pubkey = fixed_base_scalar_mul(EmbeddedCurveScalar::from_field(priv_key));
+    let pubkey_x = pubkey.x;
+    let pubkey_y = pubkey.y;
     
     // Compute input note commitment
-    let note_commitment = std::hash::pedersen([pubkey_x, pubkey_y, secret]);
+    let note_commitment = std::hash::pedersen_hash([pubkey_x, pubkey_y, secret]);
```

Comment:
I think this is so old it was pre-"the great pedersen refactor". I've used a hash here as it doesn't seem like we need the homomorphism but maybe you know better.